tests/strategies/energy_parameters/test_tank_energy_parameters.py

```diff
@@ -28,59 +28,59 @@ def setup_method(self):
         self._datetime = datetime(2023, 1, 1, 0, 0)
 
     def test_increase_tanks_temp_from_heat_energy(self):
-        self._tanks.increase_tanks_temp_from_heat_energy(1, self._datetime)
+        self._tanks.increase_tanks_temp_from_heat_energy(5000, self._datetime)
```

Comment:
@spyrostz You need 4186 J to heat up one liter of water by 1 degree. Hence, in our cases where we talk about hundrets of liters we should change to use kJ as unit for the heat demand profile in order to not have too large numbers, for the input as well as for the output. What do you think?
If you agree, I will add a ticket for this.